internal/pkg/config/regexp/regexp.go

```diff
@@ -26,6 +26,10 @@ type Regexp struct {
 	r *regexp.Regexp
 }
 
+func New(s string) *Regexp {
+	return &Regexp{r: regexp.MustCompile(s)}
```

Comment:
Out of scope, but we should consider returning an error if the regexp fails to compile and exposing that to the user in a controlled way, rather than letting it panic.

Reply:
Because as it is, I think this is likely to produce an error that will be difficult for a user to debug. It would be good to be able to say: "hey, while I was trying to unmarshal this `PackageRE` field in your yaml on line X, I ran into this wonky regex that can't be compiled. please fix" (this may be tricky to do, but I think we should move in that general direction).

Reply:
Actually, that's a good point and easy enough to add to this PR.  Done.